readme.md

```diff
@@ -605,7 +605,7 @@
 - [Speakers](https://github.com/karlhorky/awesome-speakers#readme) - Conference and meetup speakers in the programming and design community.
 - [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Fund individual programmers or the development of open source projects.
 - [Parasite](https://github.com/ecohealthalliance/awesome-parasite#readme) - Parasites and host-pathogen interactions.
-
+- [Roadmap](https://github.com/liuchong/awesome-roadmap#readme) - A curated list of roadmaps.
```

Comment:
> The entry in the Awesome list should:
Include a short description about the project/theme of the list. **It should not describe the list itself.**